docs/docs_mobile_app/polityka-cookies.md

```diff
@@ -0,0 +1,92 @@
+# POLITYKA COOKIES
+
+## Aplikacji MeldMoto
+
+**Co to jest Polityka cookies?**
+Niniejsza Polityka cookies, będąca rozszerzeniem Polityki Prywatności Aplikacji MeldMoto Warsztat oraz Polityki Prywatności Aplikacji MeldMoto Klienci, dotyczy szczegółowych zagadnień z zakresu stosowania przez nas plików cookies, kodów pikselowych oraz wdrażania innych technologii internetowych i przetwarzania danych osobowych za ich pomocą.
+
+**W niniejszej Polityce znajdą Państwo informacje o tym:**
+* czym są pliki cookies i jakie są ich rodzaje,
+* jakie technologie internetowe są stosowane w naszej Aplikacji,
+* jakie dane przetwarzamy,
+* w jaki sposób i do jakich celów wykorzystujemy dane,
+* do kogo i w jaki sposób przesyłamy dane,
+* przez jaki czas przechowujemy informacje zawarte w cookies.
+
+**Polityka cookies Aplikacji MeldMoto nie stanowi źródła obowiązków dla Użytkowników Aplikacji. Ma ona charakter wyłącznie informacyjny, nie jest umową ani regulaminem. We wszelkich kwestiach nieuregulowanych Polityką Cookies, znajduje zastosowanie:**
+
+- **Polityka Prywatności Aplikacji MeldMoto Warsztat** [https://meldmoto.github.io/terms_and_conditions/polityka-prywatnosci-warsztat](https://meldmoto.github.io/terms_and_conditions/polityka-prywatnosci-warsztat)
```

Comment:
Merge urls into headers, we don't want to see ANY url here (we should hide the fact files are located on github)